web/rootfs/defaults/meet.conf

```diff
@@ -47,8 +48,8 @@ location ~ ^/(libs|css|static|images|fonts|lang|sounds|connection_optimization|.
 
 {{ if $ENABLE_COLIBRI_WEBSOCKET }}
 # colibri (JVB) websockets
-location ~ ^/colibri-ws/([a-zA-Z0-9-\.]+)/(.*) {
-    proxy_pass http://$1:9090/colibri-ws/$1/$2$is_args$args;
```

Comment:
This is a problem if the server ID contains dots, as those will be interpreted as regex syntax.